### PR TITLE
Render unidentified issues on intake confirmation page

### PR DESCRIPTION
**Pull request by kencheeto**
Connects #7303 

### Description
- Fixes an issue where unidentified issues weren't showing up on the intake confirmation page, even though we create a Rating EP for them
- Widens the checklist so contention text is more legible 
- Displays contention_text instead of description for nonrated issues, which includes the category

Note - showing the warning alert for the unidentified issues per mock in #7303 will come in a later pr.

screen shot 2018-10-30 at 5 32 15 pm

### Acceptance Criteria
- When an intake with an unidentified issue is completed, the confirmation page shows the unidentified issue under a rating EP
- When an intake is completed, the checklist in the confirmation page displays text in a legible way
- When an intake with a nonrated issue is completed, the confirmation page includes the category when displaying the nonrated issue

### Testing Plan
1. Complete any intake with an unidentified and nonrated issue
2. On the confirmation page, confirm the points in the description

### CODE REVIEW

File: client/app/intake/pages/decisionReviewIntakeCompleted.jsx [hunk 1]
```diff
@@ -30,7 +29,7 @@ const getClaimReviewChecklistItems = (formType, requestIssues, isInformalConfere
   if (nonRatedIssues.length > 0) {
     checklist.push(<Fragment>
       <strong>A {claimReviewName} Nonrating EP is being established:</strong>
-      {nonRatedIssues.map((nri, i) => <p key={i}>Contention: {nri.description}</p>)}
+      {nonRatedIssues.map((nri, i) => <p key={i}>Contention: {nri.contentionText}</p>)}
```
Comment (kencheeto) [inline client/app/intake/pages/decisionReviewIntakeCompleted.jsx]: updating this per @leikkisa's recommendation to show the `category - description` for nonrated issues

File: client/app/styles/_intake.scss [hunk 2]
```diff
@@ -3,6 +3,11 @@
   padding-left: 0.5em;
 }
 
+.cf-app-segment .cf-intake-statusmessage-checklist {
```
Comment (kencheeto) [inline client/app/styles/_intake.scss]: the `.cf-app-segment` is needed in this rule so `.cf-intake-statusmessage-checklist` has a higher specificity than the existing .`cf-success-checklist` class.

File: client/app/intake/pages/decisionReviewIntakeCompleted.jsx (hunk 1 above)
Comment (jiahuang) [inline client/app/intake/pages/decisionReviewIntakeCompleted.jsx]: nit: should we just use `i` as a key? i'd prefer a key of `nonrated-issue-{i}` or something just to make the key more specific to this list for future-proofing.

Comment (leikkisa) [reply client/app/intake/pages/decisionReviewIntakeCompleted.jsx]: contention text works for all of the issues, (rated, unidentified, and non-rated) if you have any desire to use the same one for all of them, but it really only matters for non-rated.

Comment (kencheeto) [reply client/app/intake/pages/decisionReviewIntakeCompleted.jsx]: the contention text for unidentified issues doesn't seem helpful - it just reads `UNIDENTIFIED ISSUE - Please click \"Edit in Caseflow\" button to fix` and doesn't include the description

Comment (leikkisa) [reply client/app/intake/pages/decisionReviewIntakeCompleted.jsx]: Oh... I guess it's designed for VBMS. Good call.

**Pull request merged**